eos-tech-support/eos-convert-system

```diff
@@ -5,6 +5,13 @@ if [ ! -L /ostree ]; then
   exit 1
 fi
 
+eos-check-secureboot
+SECURE_BOOT=$?
```

Comment:
This second line won't ever run if `eos-check-secureboot` "fails" since bash is setup to exit immediately on failures. Try it this way:
```
if ! eos-check-secureboot; then
  echo "Please reboot with secure boot disabled" >&2
  exit 1
fi
```
You'll use the return code of `eos-check-secureboot` as the `if` condition, which you need with bash's `-e` enabled.